src/components/modals/login-modal/login-content.less

```diff
@@ -0,0 +1,106 @@
+.ant-btn.ant-btn-primary.login-button {
+  position: absolute;
+  left: 40.8%;
+  right: 40.8%;
+  top: 58.99%;
+  bottom: 32.58%;
+  background: #505050;
+  border-radius: 6px;
+  height: 45px;
+  width: 115px;
+}
+
+.forgot-password-text {
+  height: 44px;
+  width: 480px;
+  position: absolute;
+  left: 11.68%;
+  right: 11.52%;
+  top: 72.28%;
+  bottom: 19.48%;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 156%;
+  white-space: nowrap;
+  text-align: center;
+  color: #000000;
+}
+
+.page-link {
+  height: 14px;
+  color: black;
+  white-space: nowrap;
+  background-color: transparent;
+  border: transparent;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 156%;
+  padding: 0px 0px 0px 0px;
+}
+
+.sign-up-text {
+  position: absolute;
+  left: 11.68%;
+  right: 11.52%;
+  top: 77.34%;
+  bottom: 14.42%;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 156%;
+  align-items: center;
+  text-align: center;
+  color: #000000;
+}
+
+.title-text {
+  position: absolute;
+  left: 25.44%;
+  right: 25.6%;
+  top: 18.73%;
+  bottom: 71.54%;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 28px;
+  line-height: 140.62%;
+  align-items: center;
+  text-align: center;
+  color: #000000;
+}
+
+.ant-input {
```

Comment:
This is a dangerous ant class to override! If possible try to do this in a custom class instead